spec/jobs/searchgov_domain_destroyer_job_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+
+describe SearchgovDomainDestroyerJob do
+  subject(:perform) { SearchgovDomainDestroyerJob.perform_now(args) }
+  let(:domain) { 'www.archive.gov' }
+  let!(:searchgov_domain) {
+    SearchgovDomain.create!(domain: domain, status: 200)
+  }
+  let(:args) {
+    { searchgov_domain: searchgov_domain }
+  }
+
+  describe '#perform' do
+    it 'requires a searchgov_domain as an argument' do
+      expect{SearchgovDomainDestroyerJob.perform_now}.
+        to raise_error(ArgumentError, 'missing keyword: searchgov_domain')
+    end
+
+    it 'destroys the searchgov domain' do
+      expect(searchgov_domain).to receive(:destroy!)
+      perform
+    end
+
+    context 'when the searchgov_domain contains searchgov_urls' do
```

Comment:
This context name isn't wrong, but let's reword it to be more Rails-y:
`when the searchgov_domain has associated searchgov_url records`

"Contains" isn't generally used to describe `has many` associations.